### PR TITLE
adds 'coins-file-update' doc

### PR DESCRIPTION
**Pull request by gcharang**
@artemii235 I added a doc explaining how to update the coins file. Please review and suggest any corrections/additions

### CODE REVIEW

**Comment by sergeyboyko0791** [comment]
Hi @gcharang, I think the doc matches the functionality.
But I think the 

> Place the `coins` file in the old format in the same directory as the compiled `mm2` binary

is not a necessary, instead I suggest to mention that the user need to specify the first arg - path to the source file, second arg - path to the result file.

Thanks!

File: docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# How to update the coins file to work with the latest atomicDEX-API
+
+## Instructions
+
+- Compile the latest `mm2` binary following [these instructions](../atomicdex-setup/get-started-atomicdex.html)
+- Place the `coins` file in the old format in the same directory as the compiled `mm2` binary
```
Comment (sergeyboyko0791) [inline docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md:6]: I think it is not a necessary, instead I suggest to mention that the user need to specify the first arg - any path to the source file, second arg - any path to the result file.

Thanks!

Comment (artemii235) [reply docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md:6]: It is guide for end users, while MM2 can actually accept any path as input and output it is better to be as explicit as possible in such cases.

File: docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- Open a terminal and navigate to the directory with both the files.
+- Run the following command `./mm2 coins coins_new`
+- Output similar to the following will be shown. The "Success" message indicates that the conversion was successful
+
+```bash
+03 09:13:56, mm2:169] AtomicDEX MarketMaker f48b4ad DT 2020-07-28T17:25:18+07:00
+Success
+```
+
+- You will find a new file named `coins_new` in the same directory with the new format expected by AtomicDEX
```
Comment (artemii235) [inline docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md:16]: @gcharang Could you please add that user requires to replace `coins` with `coins_new` before attempting to run MM2?

Comment (gcharang) [reply docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md:16]: will do!

File: docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md (hunk 1 above)
Comment (gcharang) [reply docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md:6]: I am going to start documenting all the cli params of mm2. That doc will mention that mm2 can accept any valid path for the arguments. I will also add a note that mentions it in this doc.

**Comment by gcharang** [comment]
@artemii235 @sergeyboyko0791 I added the changes, please review

**Pull request merged**